wger/exercises/tests/test_exercise.py

```diff
@@ -511,6 +512,11 @@ def test_canonical_form_cache_delete(self):
         for workout_id in workout_ids:
             self.assertFalse(cache.get(cache_mapper.get_workout_canonical(workout_id)))
 
+    def test_show_all_excercise_infos(self):
+        response = self.client.get('/api/v2/exercise/special/')
+        self.assertIsInstance(response.data, list)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
 
```

Comment:
@Manorlds-Eaglespark where are the tests for getting a specific exercise and it's info?

Reply:
Thanks, @AGMETEOR , this has been resolved.